learner/tensors/test/test-B-tensor-basics.rkt

```diff
@@ -2,13 +2,13 @@
   (require rackunit)
 
   (define r0-td 3.0)
-  (define r1-td (tensor 3.0 4.0 5.0))
```

Comment:
I think it might be better to do this:   
```
  ;; for testing only.
  ;; Error checking in tensor is part of test-C-tensor-ops.rkt
  (define tensor vector)
```
for the tests in this file (similar to nested-tensors/tensors/test/test-B-tensor-basics.rkt

Reply:
That makes sense, thanks for the feedback!. Just pushed an amended commit.